pkg/quarterdeck/auth.go

```diff
@@ -97,6 +104,12 @@ func (s *Server) Register(c *gin.Context) {
 		return
 	}
 
+	// Now that the user exists in the database, send them a verification email
+	if err = s.SendVerificationEmail(user); err != nil {
+		log.Warn().Err(err).Msg("could not send verification email")
```

Comment:
```suggestion
		log.Error().Err(err).Msg("could not send verification email")
```

Let's make this an `Error` until we have retry; but in the story mentioned below we should reset this to `Warn`